gadget/params.c

```diff
@@ -425,7 +426,7 @@ void read_parameter_file(char *fname)
         All.GravitySoftening = param_get_double(ps, "GravitySoftening");
         All.GravitySofteningGas = param_get_double(ps, "GravitySofteningGas");
 
-        All.ImportBufferBoost = param_get_double(ps, "ImportBufferBoost");
+        init_treewalk(param_get_double(ps, "ImportBufferBoost"));
```

Comment:
This is a good move! We may want to gradually standarlize to objects like TreeWalkConfiguration that are used to initialize different modules.

Reply:
Really good idea!